server/core/src/main/java/dev/slimevr/VRServer.kt

```diff
@@ -308,12 +308,26 @@ class VRServer @JvmOverloads constructor(
 		queueTask { humanPoseManager.clearTrackersMounting(resetSourceName) }
 	}
 
+	fun getPauseTracking(): Boolean = humanPoseManager.getPauseTracking()
+
 	fun setPauseTracking(pauseTracking: Boolean, sourceName: String?) {
-		queueTask { humanPoseManager.setPauseTracking(pauseTracking, sourceName) }
+		queueTask {
+			humanPoseManager.setPauseTracking(pauseTracking, sourceName)
+			// Toggle trackers as they don't toggle when tracking is paused
+			if (this.getVRBridge(ISteamVRBridge::class.java)?.updateShareSettingsAutomatically() == true) {
+				RPCSettingsHandler.sendSteamVRUpdatedSettings(protocolAPI, protocolAPI.rpcHandler)
+			}
```

Comment:
should this be done before or after the set pause tracking

Reply:
After